src/component/Task/components/CompletedTask.js

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect } from 'react';
+import styled from "styled-components";
+import { getTasksFromLocalStorage, saveTasksToLocalStorage } from '../../../utils/LocalStorageUtil';
+
+
+export default function CompletedTask({ tasks, setTasks, handleDeleteTask, selectedDate }) {
+    
+    useEffect(() => {
+        const formattedDate = selectedDate.toISOString().split('T')[0]; // YYYY-MM-DD 형식
+        const loadedTasks = getTasksFromLocalStorage(formattedDate);
+
+        // Map으로 taskId와 task를 매핑
+        const newTaskMap = new Map();
+        loadedTasks.forEach(task => newTaskMap.set(task.id, task));
+
```

Comment:
```suggestion
        loadedTasks.map(task => [task.id, task]))

```
이렇게 하면 조금 더 간단해지지 않을까요..?

Reply:
요렇게만 바꿨을 때 제 task의 타입이 Map이라서 Map에는 저장이 안 될 것 같아서
피드백을 참고하여 `new Map(loadedTasks.map(task => [task.id, task]));` 요렇게 바꿔보았습니다!